cmd/manager/main.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/kudobuilder/kudo/pkg/controller/operator"
 	"github.com/kudobuilder/kudo/pkg/controller/operatorversion"
 	"github.com/kudobuilder/kudo/pkg/version"
+	webhook2 "github.com/kudobuilder/kudo/pkg/webhook"
```

Comment:
webhook2 🙈 ... should we rather do webhookServer and webhook, or calling webhook2 kudoWebhook or pretty much ... anything different than 2 :)